real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -338,46 +337,39 @@ void indicator_light_init()
 }
 
 
```

Comment:
Can we put an @brief at the beginning of each of these lighting functions like we did for the indicator light?